server/common/dao/src/test/java/org/kaaproject/kaa/server/common/dao/AbstractTest.java

```diff
@@ -747,6 +743,7 @@ protected EndpointProfileDto generateEndpointProfileDto(String appId, List<Strin
         profileDto.setSubscriptions(topicIds);
         profileDto.setEndpointKeyHash("TEST_KEY_HASH".getBytes());
         profileDto.setServerProfileBody("{\"serverTitle\": \"SERVER_TEST\"}");
+        profileDto.setEpsConfigurationHash(EPS_CONFIG_HASH);
```

Comment:
please make sure that all tests are working without setting this field. Because existing clients (v0.9 or v0.10 that will connect to new server) should be able to work without upgrade scripts.